clm_data/README.md

```diff
@@ -1,21 +1,107 @@
+- **Key Variables Related to PFTs**
+    - `PCT_NATVEG`: This variable gives the percentage of natural vegetation cover across the land units, essential for assessing non-agricultural land cover.
+    - `PCT_NAT_PFT`: Indicates the percentage composition of each PFT (16) within natural vegetation land units, providing a granular look at vegetation distribution.
 
 These PFT variables are pivotal for simulating how natural ecosystems respond to historical climatic conditions and can be used to project changes in vegetation patterns due to climatic shifts.
 
-### Data Download
-The dataset can be downloaded from the official input data repository for the Community Climate System Model:
+- **Data Download** The dataset can be downloaded from the official input data repository for the Community Climate System Model:
 [CCSM Input Data Repository](https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/surfdata_map/)
 
+### 2. `pft-physiology.c110225.nc`
+Contains physiological parameters for each of the 21 PFTs in a previous version of CLM, when vcmax25 did not depend on nitrogen content.
+
+- **Key Variables**
+    - `taulnir`: Leaf transmittance: near-IR
+    - `taulvis`: Leaf transmittance: visible
+    - `tausnir`: Stem transmittance: near-IR
+    - `tausvis`: Stem transmittance: visible
+    - `vcmx25`: Maximum rate of carboxylation
+
+- **Data Download** The dataset can be downloaded
+[here](https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/pftdata/pft-physiology.c110225.nc)
+
+### 3. `clm5_params.c171117.nc`
+Contains various parameters required by the CLM5 model.
+
+- **Key Variables**
+    - `medlynslope`: Medlyn slope of conductance-photosynthesis relationship
+    - `medlynintercept`: Medlyn intercept of conductance-photosynthesis relationship
+
+- **Data Download** The dataset can be downloaded
+[here](https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/paramdata/clm5_params.c171117.nc)
+
+
+## Output Files
+
+### 1. `dominant_PFT_map.nc`
+Contains the dominant PFT for each grid cell on the global grid.
+- **Purpose**: Used as an input file for mapping the physiological parameters to the global grid based on the dominant PFT.
+
+### 2. `vegetation_properties_map.nc`
+Contains the mapped vegetation properties for each grid cell.
+- **Contents**:
+  - `medlynslope(lat, lon)`: Medlyn slope of conductance-photosynthesis relationship (kPa^0.5)
+  - `medlynintercept(lat, lon)`: Medlyn intercept of conductance-photosynthesis relationship (umol m^-2 s^-1)
+  - `taulnir(lat, lon)`: Leaf transmittance: near-IR (fraction)
+  - `taulvis(lat, lon)`: Leaf transmittance: visible (fraction)
+  - `tausnir(lat, lon)`: Stem transmittance: near-IR (fraction)
+  - `tausvis(lat, lon)`: Stem transmittance: visible (fraction)
+  - `vcmx25(lat, lon)`: Maximum rate of carboxylation (umol CO2/m**2/s)
```

Comment:
Typically (in the CF conventions), `lon` is the first dimension, and `lat` the second

Reply:
No, it is not. Please refer to: https://docs.google.com/document/d/1sWfm3UWu3is-ZykZXyobAD1Vp4Vc7ccpRB0abwMMJlE/edit?usp=sharing